javalang/tokenizer.py

```diff
@@ -579,10 +582,12 @@ def error(self, message, char=None):
 
         message = u'%s at "%s", line %s: %s' % (message, char, line_number, line)
 
-        raise LexerError(message)
+        if not self.ignore_errors:
```

Comment:
I think it could be helpful to keep track of these errors even if we're not raising them them when `ignore_errors=True`. I'd suggest we add each error to a new `self.errors` list (regardless of the `ignore_errors` value) and raise the error only when `ignore_errors=False`.

Reply:
Done.